tools/packaging/static-build/qemu/build-static-qemu-cc.sh

```diff
@@ -7,29 +7,88 @@
 set -o errexit
 set -o nounset
 set -o pipefail
+set -x
 
 script_dir="$(cd "$(dirname "${BASH_SOURCE[0]}")" && pwd)"
 
 source "${script_dir}/../../scripts/lib.sh"
 
-qemu_repo="${qemu_repo:-}"
-qemu_version="${qemu_version:-}"
-tee="${tee:-}"
+export qemu_repo="${qemu_repo:-}"
+export qemu_version="${qemu_version:-}"
+export qemu_cc_latest_build_url="${jenkins_url}/job/kata-containers-2.0-qemu-cc-$(uname -m)/${cached_artifacts_path}"
+export katacontainers_repo="${katacontainers_repo:=github.com/kata-containers/kata-containers}"
+export katacontainers_repo_dir="${GOPATH}/src/${katacontainers_repo}"
+export pkg_dir="${katacontainers_repo_dir}/tools/packaging/static-build"
+export qemu_cc_tarball_directory="${pkg_dir}/build/cc-qemu/builddir"
+export tarball_name="kata-static-qemu-cc.tar.gz"
+export tee="${tee:-}"
 
 export prefix="/opt/confidential-containers/"
 
-if [ -z "$qemu_repo" ]; then
-	info "Get qemu information from runtime versions.yaml"
-	qemu_url=$(get_from_kata_deps "assets.hypervisor.qemu.url")
-	[ -n "$qemu_url" ] || die "failed to get qemu url"
-	qemu_repo="${qemu_url}.git"
-fi
-[ -n "$qemu_repo" ] || die "failed to get qemu repo"
+get_qemu_information() {
+	if [ -z "$qemu_repo" ]; then
+		info "Get qemu information from runtime versions.yaml"
+		export qemu_url=$(get_from_kata_deps "assets.hypervisor.qemu.url")
+		[ -n "$qemu_url" ] || die "failed to get qemu url"
+		export qemu_repo="${qemu_url}.git"
+	fi
 
-[ -n "$qemu_version" ] || qemu_version=$(get_from_kata_deps "assets.hypervisor.qemu.version")
-[ -n "$qemu_version" ] || die "failed to get qemu version"
+	[ -n "$qemu_repo" ] || die "failed to get qemu repo"
+	[ -n "$qemu_version" ] || export qemu_version=$(get_from_kata_deps "assets.hypervisor.qemu.version")
+	[ -n "$qemu_version" ] || die "failed to get qemu version"
+}
 
+calc_qemu_files_sha256sum() {
+	local files="${pkg_dir}/qemu"
```

Comment:
This will expand to `${katacontainers_repo_dir}/tools/packaging/static-build/qemu`,

I think we want to actually calculate the sha256sum of `${katacontainers_repo_dir}/tools/packaging/qemu/patches/6.2.x`, for instance.

The reason for that being that `${katacontainers_repo_dir}/tools/packaging/static-build/qemu` only contains the scripts used to build QEMU, and not the patches applied to a specific version.

This is causing the check on the CI to return that the sha256sum is empty, as there's no files found in the `sha256sum_from_files` function.